### ISSUE TITLE
指标计算除了token_acc，还可以添加自定义的指标i吗

### ISSUE THREAD

**Issue by zzk2021** [modelscope/ms-swift#7291]
{'loss': 0.27710536, 'grad_norm': 2.0904994, 'learning_rate': 6.16e-06, 'token_acc': 0.89303676, 'epoch': 0.59, 'global_step/max_steps': '5295/9017', 'percentage': '58.72%', 'elapsed_time': '17h 16m 32s', 'remaining_time': '12h 8m 36s', 'memory(GiB)': 45.9, 'train_speed(iter/s)': 0.085139}

**Comment by Jintao-Huang** [comment]
please modify Seq2SeqTrainer compute_loss function
